src/main/java/detail/TableDataSet.java

```diff
@@ -0,0 +1,29 @@
+package detail;
+
+import tech.tablesaw.api.Table;
+
+import java.io.File;
+
+/**
+ * Wrapper to the Tablesaw Table class
+ */
+public class TableDataSet implements IDataSet {
```

Comment:
why is this class so different than MapDataSet?

Reply:
you mean the ```MapDataContainer```? we don't have ```MapDataSet```

Reply:
sorry that's what I meant. don't we need a `MapDataSet` class?

Reply:
hmmm I am not sure yet....